README.md

```diff
@@ -44,35 +42,59 @@
   </ul>
 </details>
 
-<a name="key-features"></a>
-
 - **[Channge the content color]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<a name="live-demo"></a>
+## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://AbednegoKibitok/github.io/set-up-and-mobile-first)
+Not applicable
```

Comment:
Kindly, I suggest you remove the live demo section if you don't deploy your project yet. 😄